README.md

```diff
@@ -21,6 +21,12 @@ Implementing Savvy Widget is easy. Just copy-and-paste a few lines of code:
 </script>
 ```
 
+## Attribution & Tracking
+
+In order to get credit for your usage, you MUST set `data-url-tracking-params` to the value assigned by your partner success manager.
```

Comment:
In this case it is `urlTrackingParams` in the `Savvy.configure`.

Reply:
Fixed.